src/font/lv_font_loader.c

```diff
@@ -85,15 +85,16 @@ static unsigned int read_bits(bit_iterator_t * it, int n_bits, lv_fs_res_t * res
  */
 lv_font_t * lv_font_load(const char * font_name)
 {
-    lv_fs_file_t file;
-    lv_fs_res_t res = lv_fs_open(&file, font_name, LV_FS_MODE_RD);
+    lv_fs_file_t * file = lv_mem_alloc(sizeof(lv_fs_file_t));
```

Comment:
Question, the file open function probably deals with a potential alloc returning null, but it would not better to assert the alloc result to tell the user the exact point of crashing? It would be helpful on debug.

Reply:
Thanks, updated!